bucoffea/gen/lheVProcessor.py

```diff
@@ -149,6 +158,9 @@ def process(self, df):
                                     )
 
             mask_monojet = monojet_sel.all(*monojet_sel.names)
+            # Apply manual DR>0.4 Cut on GJets sample
+            if is_lo_g(dataset) or is_nlo_g(dataset) or is_lo_g_ewk(dataset) or is_nlo_g_ewk(dataset):
+                mask_monojet = mask_monojet * (df[f'minDR_lhe_parton_photon']>0.4)
```

Comment:
Instead of manually changing the mask, can't we rather do something like:
```python
if photon:
	monojet_sel.add("lhedr", DR > 0.4)
```
inside of the `monojet_selection` function? Also, same for VBF

Reply:
I just thought this is less lines of code. Otherwise we need to add it to both monojet and VBF selections and add DR as an input to the selector